### ISSUE TITLE
[Bug] Exception when downloading Multi Video Post

### ISSUE THREAD

**Issue by MohamedIM** [althonos/InstaLooter#42]
An exception is thrown when downloading multi video post
```python
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/site-packages/instaLooter/worker.py", line 44, in run
    self._download_video(media)
  File "/usr/lib/python3.6/site-packages/instaLooter/worker.py", line 120, in _download_video
    video_url = data["video_url"]
KeyError: 'video_url'
```

**Comment by althonos** [comment]
Do you have a link to the post that triggers this exception ? It would be easier for me to debug. Thanks 😄

**Comment by MohamedIM** [comment]
For example this post from Instagram account: https://www.instagram.com/p/BRW-j_dBI6F/
Also I think it's a good exercise to test with instagram account it self as I found other types of exception thrown when I tried to download full videos and images from "instagram" account e.g. this error using command ``` instaLooter -v -m instagram instagram``` :
```python 
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/site-packages/instaLooter/worker.py", line 46, in run
    self._download_photo(media)
  File "/usr/lib/python3.6/site-packages/instaLooter/worker.py", line 92, in _download_photo
    self._add_metadata(photo_name, media)
  File "/usr/lib/python3.6/site-packages/instaLooter/worker.py", line 75, in _add_metadata
    piexif.ExifIFD.DateTimeOriginal: datetime.datetime.fromtimestamp(metadata['date']).isoformat(),
KeyError: 'date'
```

**Comment by Bangaio64** [comment]
I found some weird exception in the new version, but I'm not sure what causes it. Seems to happen randomly.

`Traceback (most recent call last):
  File "c:\program files (x86)\python36-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files (x86)\python36-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files (x86)\Python36-32\Scripts\instaLooter.exe\__main__.py", line 9, in <module>
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\__main__.py", line 130, in main
    new_only=args['--new'],
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\core.py", line 388, in download
    new_only=new_only)
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\core.py", line 445, in _fill_media_queue
    for media in self.medias(media_count=media_count, with_pbar=with_pbar, timeframe=timeframe):
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\core.py", line 286, in _timeless_medias
    for page in self.pages(media_count=media_count, with_pbar=with_pbar):
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\core.py", line 258, in pages
    warnings.warn("Profile {} is private, retry after logging in.".format(self.target))
  File "c:\program files (x86)\python36-32\lib\warnings.py", line 99, in _showwarnmsg
    msg.file, msg.line)
  File "c:\program files (x86)\python36-32\lib\site-packages\instaLooter\utils.py", line 64, in warn_windows
    "{msg}").format(t=time.localtime(), msg=message), file=sys.stderr)
NameError: name 'time' is not defined`

**Comment by althonos** [comment]
@Bangaio64: seems like I made a typo in the `warn_windows` version, so only a Windows user can see that. I'm opening another issue for this.

**Comment by althonos** [comment]
@MohamedIM: I'll add a test for the `instagram` profile. For the two exceptions, I think I know where they do come from. I'll try to patch that quickly.

**Comment by althonos** [comment]
Should be fixed in new version (v0.7.2). Closing. Don't hesitate to report if it still doesn't work.
